getting-started/scaling-out.md

```diff
@@ -0,0 +1,128 @@
+# Scaling out using Distributed Hypertables
```

Comment:
Agree with Erik.  Maybe start with a brief paragraph that then has a link to this section for more info? 
 https://github.com/timescale/docs.timescale.com-content/pull/169/files

(In fact, that PR includes first paragraph or so that could just be summarized here)